web/src/views/Home.vue

```diff
@@ -35,24 +47,35 @@ import {
 const JUPYTER_ROOT = 'https://hub.dandiarchive.org';
 
 const actionKeys = [
-   {
+  {
     for: ['item'],
     name: 'Open JupyterLab',
     icon: 'mdi-language-python',
     color: 'primary',
     handler() {
       window.open(`${JUPYTER_ROOT}`, '_blank');
     },
-   },
+  },
   ...DefaultActionKeys.slice(1),
 ];
 
 export default {
   components: { GirderDataDetails, GirderFileManager },
   computed: {
+    isDandiset() {
+      return (
+        !!this.location
+        && !!this.location.meta
+        && !!this.location.meta.dandiset
```

Comment:
To save on both punctuation and time, we can just do

```javascript
!!(this.location && this.location.meta && this.location.meta.dandiset)
```

And hold out hope for the [null propagation operator](https://ponyfoo.com/articles/null-propagation-operator).

Reply:
Good point. Can't wait to be using the "Elvis" operator.